test/driver.spec.js

```diff
@@ -84,6 +90,33 @@ describe('did:key method driver', () => {
         .equal('zDnaeucDGfhXHoJVqot3p21RuupNJ2fZrs8Lb1GV83VnSo2jR');
     });
 
+    it('should get the DID Document for a X25519-based DID', async () => {
+      const did = 'did:key:z6LSbysY2xFMRpGMhb7tFTLMpeuPRaqaWM1yECx2AtzE3KCc';
+      const keyId =
+        `${did}#z6LSbysY2xFMRpGMhb7tFTLMpeuPRaqaWM1yECx2AtzE3KCc`;
+      const didKeyDriverKak = driver();
+
+      didKeyDriverKak.use({
+        multibaseMultikeyHeader: 'z6LS',
+        // FIXME: Use EcdsaMultikey instead once X25519KeyAgreementKey support
+        // is added in EcdsaMultikey
```

Comment:
ECDSA would never work with X25519 because they are based on different elliptic curves (P-256 and Curve25519, respectively). The ECDSA Multikey key material would be the same whether it is used as a key agreement key or for signature verification. I'll file an issue on this.

```suggestion
```

Reply:
Removed all FIXME comments [1140351](https://github.com/digitalbazaar/did-method-key/pull/58/commits/1140351475e0c49ea903a5a1938488d7fd7c39da)